### PR TITLE
Lesson 6

### PR DESCRIPTION
**Pull request by YuliyaCher**
1. Made a layout of the top block (except for search strings) + started making a mobile version.
2. Made a mobile version of footer.

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -6,15 +6,60 @@
...
     <link rel="stylesheet" href="./style.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
-        rel="stylesheet"/>
+        rel="stylesheet" />
     <title>Document</title>
 </head>
 
 <body>
```
Comment (ghost) [inline Triphouse-project/index.html:15]: 2021-04-16_00h32_26
Wrap header main footer into some div with {max-width: 1440px, margin: 0px auto}

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -6,15 +6,60 @@
...
     <header>
+        <div class="wrapper">
+            <div class="wrapper_desktop">
+                <div class="navigation">
+                    <div class="navigation_logo">
+                        <img src="./img/logo_vector.png">
+                    </div>
+                    <div class="navigation_item">
```
Comment (ghost) [inline Triphouse-project/index.html:23]: Use `ul` instead `div`.

Comment (YuliyaCher) [reply Triphouse-project/index.html:23]: corrected

File: Triphouse-project/index.html (hunk 1 above)
Comment (YuliyaCher) [reply Triphouse-project/index.html:15]: done

**Pull request merged**